must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -159,4 +162,15 @@ for ns in $namespaces; do
         { timeout 120 oc debug nodes/"${node}" -- bash -c "test -f /host/var/lib/rook/log/${ns}/ceph-volume.log && cat /host/var/lib/rook/log/${ns}/ceph-volume.log" > "${NODE_OUTPUT_DIR}"/ceph-volume.log; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
     done
     oc delete -f pod_helper.yaml
+
+    # Collecting ceph prepare volume logs
```

Comment:
The comment needs to be changed to capturing crash dumps.

Reply:
done